framework/base/helpers/dom.js

```diff
@@ -3,3 +3,21 @@ export const queryAll = (selector, context = document) =>
   [].slice.call(context.querySelectorAll(selector));
 
 export default queryAll;
+
+export const hasClass = (element, cssClass) =>
```

Comment:
the `hasClass` is actually a helper for the other 2 functions I added hoping to make some common tasks easier. I really don't mind removing them?